vendor/bat-native-ledger/src/bat/ledger/internal/uphold/uphold_unittest.cc

```diff
@@ -39,7 +39,7 @@ class UpholdTest : public testing::Test {
 };
 
 TEST_F(UpholdTest, FetchBalanceConnectedWallet) {
-  EXPECT_CALL(*mock_ledger_impl_, LoadURL(_, _, _, _, _, _)).Times(0);
+  EXPECT_CALL(*mock_ledger_client_, LoadURL(_, _, _, _, _, _)).Times(0);
```

Comment:
So here, because this is not a public member of LedgerImpl, we have to mock the client level, hence the changes above (otherwise this test - which was originally passing - fails).

Reply:
`LoadURL` is public method in ledger_impl.h https://github.com/brave/brave-core/blob/ab783ba8688d225aac90dd4847247958a41c5ab3/vendor/bat-native-ledger/src/bat/ledger/internal/ledger_impl.h#L229

Reply:
😕 clearly having a brain-melt there! 

However, the LoadURL changes are required.

Reply:
still not sure that I understand this change. It goes through `mock_ledger_impl_`, something else should be going on

Reply:
Happy to change - though the changes were needed due to the call paths being invoked by the existing code, which I/we shouldn't be changing to allow a unit test to pass (of course). So the mock itself had to be changed to accommodate the code. If this can be done in a different / better way - I'm all ears 👍

Reply:
- [ ] Ticket for the mock issues discussed in this PR